editoast/src/valkey_utils.rs

```diff
@@ -11,16 +14,13 @@ use deadpool_redis::redis::RedisError;
 use deadpool_redis::redis::RedisFuture;
 use deadpool_redis::redis::ToRedisArgs;
 use deadpool_redis::redis::Value;
-use deadpool_redis::Config;
-use deadpool_redis::Connection;
-use deadpool_redis::Pool;
-use deadpool_redis::PoolError;
-use deadpool_redis::Runtime;
-use futures::future;
+use deadpool_redis::redis::aio::ConnectionLike;
+use deadpool_redis::redis::cmd;
 use futures::FutureExt;
-use serde::de::DeserializeOwned;
+use futures::future;
 use serde::Serialize;
-use tracing::{debug, span, trace, Level};
+use serde::de::DeserializeOwned;
+use tracing::{Level, debug, span, trace};
```

Comment:
nitpick: Since it's part of the diff, can you flatten that `use`, please?

Reply:
Yep, do we have a command line to flatten everything?

Reply:
Not that I'm aware of, but you can have a `rustfmt.toml` somewhere with `imports_granularity = "item"` and `cargo +nightly fmt` if I'm not mistaken

https://rust-lang.github.io/rustfmt/?version=v1.8.0&search=granula#imports_granularity

Reply:
Note, we did not enforce that in the CI, because it would also impose this on all developers and that functionality depends on `nightly` so it felt too much to impose on everyone.